[68.920612335] 2019-05-13 18:21:41,670 - INFO - launcher: Starting sanity checks
[69.921493530] 2019-05-13 18:21:41,671 - INFO - launcher: Checking for Python 3.5+
[70.925235748] 2019-05-13 18:21:41,672 - INFO - launcher: Checking console encoding
[70.925235748] 2019-05-13 18:21:41,672 - INFO - launcher: Setting console encoding to UTF-8
[70.925235748] 2019-05-13 18:21:41,672 - INFO - launcher: Ensuring we're in the right environment
[74.925661087] 2019-05-13 18:21:41,676 - INFO - launcher: Adding local bins/ folder to path
[4109.215021133] 2019-05-13 18:21:45,710 - INFO - launcher: Required checks passed.
[4110.215902328] 2019-05-13 18:21:45,711 - INFO - launcher: Optional checks passed.
[4110.715866089] 2019-05-13 18:21:45,712 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[7.3280034065246582] 2019-05-13 18:21:48,929 - WARNING - musicbot.config | In config.py::MainThread(2128), line 118 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[7.3285045623779297] 2019-05-13 18:21:48,929 - INFO - musicbot.config | In config.py::MainThread(2128), line 129 in run_checks: Using i18n: config/i18n/en.json
[7.3300051689147949] 2019-05-13 18:21:48,931 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 226 in _setup_logging: Set logging level to INFO
[7.4991254806518555] 2019-05-13 18:21:49,100 - DEBUG - musicbot.json | In json.py::MainThread(2128), line 8 in __init__: Init JSON obj with config/i18n/en.json
[7.7713184356689453] 2019-05-13 18:21:49,372 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 91 in __init__: Starting MusicBot release-260419
[7.8498752117156982] 2019-05-13 18:21:49,451 - WARNING - musicbot.bot | In bot.py::MainThread(2128), line 94 in __init__: Autoplaylist is empty, disabling.
[11.1277058124542236] 2019-05-13 18:21:52,729 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 942 in on_ready: Connection established, ready to go.
[11.1282064914703369] 2019-05-13 18:21:52,729 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 340 in _cache_app_info: Caching app info
[11.3538677692413330] 2019-05-13 18:21:52,955 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 764 in _scheck_ensure_env: Ensuring data folders exist
[11.3553676605224609] 2019-05-13 18:21:52,956 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 780 in _scheck_server_permissions: Checking server permissions
[11.3558688163757324] 2019-05-13 18:21:52,957 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 784 in _scheck_autoplaylist: Auditing autoplaylist
[11.3558688163757324] 2019-05-13 18:21:52,957 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 788 in _scheck_configs: Validating config
[11.3558688163757324] 2019-05-13 18:21:52,957 - DEBUG - musicbot.config | In config.py::MainThread(2128), line 226 in async_validate: Validating options...
[11.3563687801361084] 2019-05-13 18:21:52,957 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 791 in _scheck_configs: Validating permissions config
[11.3563687801361084] 2019-05-13 18:21:52,957 - DEBUG - musicbot.permissions | In permissions.py::MainThread(2128), line 94 in async_validate: Validating permissions...
[11.3563687801361084] 2019-05-13 18:21:52,957 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 959 in on_ready: Connected: 576264998746325032/우리동네 음악대장#7208
[11.3573694229125977] 2019-05-13 18:21:52,958 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 967 in on_ready: Owner:     343761769270870026/Willtriti#4210

[11.3583703041076660] 2019-05-13 18:21:52,959 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 970 in on_ready: Guild List:
[11.3588705062866211] 2019-05-13 18:21:52,960 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 974 in on_ready:  - 대신중 ㅄ들
[11.3593709468841553] 2019-05-13 18:21:52,960 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 1030 in on_ready: Not bound to any text channels
[11.3598711490631104] 2019-05-13 18:21:52,961 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 1056 in on_ready: Not autojoining any voice channels
[11.3608717918395996] 2019-05-13 18:21:52,962 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 270 in _join_startup_channels: Found owner in "=====  로    비  ====="
[11.3623733520507812] 2019-05-13 18:21:52,963 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 279 in _join_startup_channels: Attempting to join 대신중 ㅄ들/=====  로    비  =====
[12.2025084495544434] 2019-05-13 18:21:53,803 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 724 in deserialize_queue: Deserializing queue for 386103174014042113
[12.7045104503631592] 2019-05-13 18:21:54,305 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 423 in get_player: Created player via deserialization for guild 386103174014042113 with 0 entries
[12.7050228118896484] 2019-05-13 18:21:54,306 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 295 in _join_startup_channels: Joined 대신중 ㅄ들/=====  로    비  =====
[12.7065091133117676] 2019-05-13 18:21:54,307 - DEBUG - musicbot.bot | In bot.py::MainThread(2128), line 523 in on_player_stop: Running on_player_stop
[1238.4442443847656250] 2019-05-13 18:42:20,045 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 2668 in on_message: 343761769270870026/Willtriti#4210: /help
[1304.4355838298797607] 2019-05-13 18:43:26,036 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 2668 in on_message: 343761769270870026/Willtriti#4210: /setname
[4778.5307974815368652] 2019-05-13 19:41:20,132 - INFO - musicbot.bot | In bot.py::MainThread(2128), line 2668 in on_message: 343761769270870026/Willtriti#4210: /help
